examples/tutorials/curation/plot_2_train_a_model.py

```diff
@@ -0,0 +1,125 @@
+"""
+Training a model for automated curation
+=============================
+
+If the pretrained models do not give satisfactory performance on your data, it is easy to train your own classifier using SpikeInterface.
+"""
+import warnings
+warnings.filterwarnings("ignore")
+from pathlib import Path
+import numpy as np
+import pandas as pd
+import matplotlib.pyplot as plt
+
+import spikeinterface.core as si
+import spikeinterface.curation as sc
+import spikeinterface.widgets as sw
+
+
+# Note, you can set the number of cores you use using e.g.
+# si.set_global_job_kwargs(n_jobs = 8)
+
+##############################################################################
+# Step 1: Generate and label data
+# -------------------------------
+#
+# For this tutorial, we will use simulated data to create ``recording`` ``sorting`` objects. We'll
+# create two sorting objects: :code:`sorting_1` is coupled to the real recording, so will contain good
+# units; :code:`sorting_2` is uncoupled, so should produce noise. We'll combine the two into one sorting
+# object using :code:`si.aggregate_units`.
+#
+# (When making your own model, you should
```

Comment:
This is a really cool section but I got a bit confused during it. I would reorder slightly and add some examples to make it super clear how you could use it in real life. For example:

1) We will create the test dataset as above. Then, `plot_unit_templates` to confirm our test data is setup properly.
2) In real life, we would have a recording and sorting that is our data. We would then compute quality metrics and manually curate a set of labels. Now, create the sorting analyzer and compute metrics
3) mention again in real life we would now manually curate the data. and tada! here's some labels we got from this.